documentation/configure.markdown

```diff
@@ -180,6 +180,8 @@ When this is set to true, debugging information will be written to `<vardir>/deb
 
 ### `url-prefix`
 
+> **Deprecated:** This setting will be removed in the future and url-prefixes should be configured using the `web-router-service`.
```

Comment:
Anything but the simplest configuration of the web-router-service is hard to do painful because it's nested https://github.com/puppetlabs/trapperkeeper-webserver-jetty9/blob/master/doc/webrouting-config.md. I doubt anyone will do anything more complicated than the simplest configuration atm since there isn't really a need, but it seems like our plans for the web-router-service, mounting metrics next to pdb next to "foo", would require some of that fine tuning. All this to say, I think it should probably be done in hocon if we think that's ok.

Is the work to split out metrics etc. to their own services planned for 3.0.0?